resources/startup.sh

```diff
@@ -4,129 +4,133 @@ set -o nounset
 set -o pipefail
 
 # variables
-ADMUSR="admin"
-ADMDEFAULTPW="admin123"
-ADMINGROUP=$(doguctl config --global admin_group)
-DOMAIN=$(doguctl config --global domain)
-MAIL_ADDRESS=$(doguctl config -d "nexus@${DOMAIN}" --global mail_address)
-FQDN=$(doguctl config --global fqdn)
-
-function set_random_admin_password {
-  ADMPW=$(doguctl random)
-  curl -s --retry 3 --retry-delay 10 -X POST -H "Content-Type: application/json" -d "{data:{userId:\"$ADMUSR\",newPassword:\"$ADMPW\"}}" --insecure 'http://127.0.0.1:8081/nexus/service/local/users_setpw' -u "$ADMUSR":"$ADMDEFAULTPW"
-  doguctl config -e "admin_password" "${ADMPW}"
-  echo "${ADMPW}"
+ADMINUSER="admin"
+export ADMINDEFAULTPASSWORD="admin123"
+NEXUS_DATA_DIR=/var/lib/nexus
+# credentials for nexus-scripting tool
+export NEXUS_URL="http://localhost:8081/nexus"
+export NEXUS_USER=${ADMINUSER}
+export NEXUS_PASSWORD=${ADMINDEFAULTPASSWORD}
+# create random admin password
+NEWADMINPASSWORD=$(doguctl random)
```

Comment:
Why generate new password on every startup?

Reply:
Not really necessary at subsequent start, but the NEWADMINPASSWORD variable has to exist so that nexusConfParameters.json.tpl can be rendered.